pkg/tsdb/v3iotsdb_integration_test.go

```diff
@@ -0,0 +1,311 @@
+// +build integration
+
+/*
+Copyright 2018 Iguazio Systems Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License") with
+an addition restriction as set forth herein. You may not use this
+file except in compliance with the License. You may obtain a copy of
+the License at http://www.apache.org/licenses/LICENSE-2.0.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License.
+
+In addition, you may not use the software for any purposes that are
+illegal under applicable law, and the grant of the foregoing license
+under the Apache 2.0 license is conditioned upon your compliance with
+such restriction.
+*/
+
+package tsdb_test
+
+import (
+	"fmt"
+	"github.com/v3io/v3io-go-http"
+	"github.com/v3io/v3io-tsdb/pkg/config"
+	. "github.com/v3io/v3io-tsdb/pkg/tsdb"
+	"github.com/v3io/v3io-tsdb/pkg/tsdb/tsdbtest"
+	"github.com/v3io/v3io-tsdb/pkg/utils"
+	"path/filepath"
+	"reflect"
+	"testing"
+)
+
+func TestIngestData(t *testing.T) {
+	v3ioConfig, err := config.LoadConfig(filepath.Join("../../", config.DefaultConfigurationFileName))
```

Comment:
Look at the src/github.com/v3io/v3io-tsdb/test/benchmark/common/config.go
Lets unify the code